integration-tests/opentelemetry-spi/src/main/java/io/quarkus/it/opentelemetry/spi/ExporterResource.java

```diff
@@ -35,6 +41,13 @@ public List<SpanData> export() {
                 .collect(Collectors.toList());
     }
 
+    @GET
+    @Path("/export/propagation")
+    public List<String> exportPropagation() throws NoSuchFieldException, IllegalAccessException {
+        TextMapPropagator textMapPropagator = openTelemetry.getPropagators().getTextMapPropagator();
+        return new ArrayList<>(textMapPropagator.fields());
```

Comment:
So no reflection needed at the end?
(since afair, previous version of this had it ... )

Reply:
Yes, the reflection was not working well with the devmode test. This achieves the same validation.